lib/ddtrace/contrib/elasticsearch/patcher.rb

```diff
@@ -25,6 +40,63 @@ def patch
           @patched
         end
 
+        # rubocop:disable Metrics/MethodLength
+        def patch_elasticsearch_transport_client
```

Comment:
It might be good to make sure this works with a few of the [major transport implementations](https://github.com/elastic/elasticsearch-ruby/tree/master/elasticsearch-transport#transport-implementations) .  e.g. (patron and faraday)